src/SFA.DAS.EmployerIncentives.Commands/CompleteEarningsCalculation/CompleteEarningsCalculationCommandValidator.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading.Tasks;
```

Comment:
This validator can also be replaced with a NullValidator as Kevin suggested above ^^

Reply:
Good luck getting the NullValidator to work!  Part of the DI framework within functions doesn't support it.

Reply:
The file has been removed